From: Jan Engelhardt <jengelh@inai.de>
Date: 2013-06-14 13:55:00.148814697 +0200

Work around compile abort with libdb-6.0

BDB has removed RPC support (which was not implemented anyway;
it [set_rpc_server] always returned EOPNOTSUPP even in libdb-4.8).

---
 clisp-536a48a9/modules/berkeley-db/bdb.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- clisp-536a48a9/modules/berkeley-db/bdb.c
+++ clisp-536a48a9/modules/berkeley-db/bdb.c	2016-11-29 11:58:23.833524665 +0000
@@ -60,6 +60,7 @@ static void vecout (unsigned char* v, in
    this is a macro and not a function for NO_UNICODE */
 #define asciz_to_string0(a,e) (a ? asciz_to_string(a,e) : NIL)
 
+#include <errno.h>
 #include <db.h>
 
 /* http://www.oracle.com/technology/products/berkeley-db/db/update/4.4.20/if.4.4.20.html
@@ -2216,7 +2217,7 @@ DEFUN(BDB:DB-COMPACT, db &key TRANSACTIO
   c_data.compact_timeout = timeout;
   c_data.compact_pages = pages;
   SYSCALL(db->compact,(db,txn,pstart,pstop,&c_data,flags,&end));
-  pushSTACK(uint32_to_I(c_data.compact_empty_buckets));
+/*pushSTACK(uint32_to_I(c_data.compact_empty_buckets));*/
   pushSTACK(uint32_to_I(c_data.compact_pages_free));
   pushSTACK(uint32_to_I(c_data.compact_pages_examine));
   pushSTACK(uint32_to_I(c_data.compact_levels));
